util-scripts/sspl_update

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+set -e -u -o pipefail
+
+PRODUCT_NAME='LDR_R1'
+PRODUCT_FAMILY='cortx'
+
+[ -f /var/cortx/sspl/data/state.txt ] && {
+    if [[ $(cat /var/cortx/sspl/data/state.txt ) == "state=active" ]]; 
+    then
+        # TODO Failover to other node, avoid pcs
+        echo "Doing failover"
+    fi
+}
+
+# Disable sspl service
+pcs resource disable sspl-master
```

Comment:
Comments from Mandar which we can think of incorporating

>[9:43 PM] Mandar Sawant
    Yeah, but when you update the rpms on the node the resources should failover to peer node or you want to update just the sspl on that node? even in case of the latter, we can ban the sspl resource on that node so that it failsover to other node.

> ​[9:50 PM] Mandar Sawant
move the resource to other node,
	pcs resource move sspl srvnode-[1/2] --master
	ban the resource on the node
	pcs resource ban sspl srvnode-[1/2] --master